package-lock.json

```diff
@@ -41,7 +41,7 @@
       "resolved": "https://registry.npmjs.org/@etpinard/gl-text/-/gl-text-1.1.6.tgz",
       "integrity": "sha512-sN007FwlqdSKJt2/cnGZu3jsAN7G4R/wxk/D6ZivPuQtrwJ42B68iuAqysJPgFepUTAsDRtGAOd1U7tZxfDJwA==",
       "requires": {
-        "color-normalize": "1.1.0",
+        "color-normalize": "1.3.0",
```

Comment:
@dy I just noticed that we're using multiple versions of `color-normalize` in our bundle:

![image](https://user-images.githubusercontent.com/6675409/43596927-a0357978-964e-11e8-8de2-d607e3a264b5.png)

Reply:
similarly for `color-rgba`:

![image](https://user-images.githubusercontent.com/6675409/43596966-b4d801e8-964e-11e8-84fe-881ccbf85dc6.png)

Reply:
Can `regl-error2d`, `regl-line2d` and `regl-scatter2d` use `color-normalize@1.3.0`?